week-4/implement/is-prime.test.js

```diff
@@ -1,3 +1,37 @@
 // Given a positive integer num,
 // When the isPrime function is called with num as input,
 // Then it should return a boolean representing whether the num is prime
+
+//................................................................
+
+// ANSWER
+
+function isPrime(num) {
+    if (num <= 1) {
+      return false; // 0 and 1 are not prime numbers
+    }
+  
+    for (let i = 2; i <= Math.sqrt(num); i++) {
```

Comment:
This is completely true. As a further improvement, create a variable for `Math.sqrt(num)` to improve readability. (Like you did before)